docs/_templates/overrides/metpy.calc.rst

```diff
@@ -93,6 +93,7 @@ Soundings
       storm_relative_helicity
       supercell_composite
       surface_based_cape_cin
+      downdraft_cape
```

Comment:
Can you move this to keep the list alphabetized?

Reply:
I thought these were just grouped into 'similar categories of parameters' and I didn't even realize these were alphabetized ;P